src/webhook/validation/dynakube/dynakube_name.go

```diff
@@ -0,0 +1,26 @@
+package dynakube
+
+import (
+	"unicode"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1/dynakube"
+)
+
+const (
+	errorDigitInName = `The DynaKube's specification has an invalid name: It starts with a digit.
+	`
+)
+
+func nameStartsWithDigit(dv *dynakubeValidator, dynakube *dynatracev1beta1.DynaKube) string {
+	dynakubeName := dynakube.Name
```

Comment:
just curious, why don't we validate in same way as k8s does?

https://github.com/kubernetes/kubernetes/blob/f7dacb63cc0b0e79450b2f6d97b1c6e6c46070bd/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L236-L245C2
```
func IsDNS1035Label(value string) []string {
        var errs []string
        if len(value) > DNS1035LabelMaxLength {
                errs = append(errs, MaxLenError(DNS1035LabelMaxLength))
        }
        if !dns1035LabelRegexp.MatchString(value) {
                errs = append(errs, RegexError(dns1035LabelErrMsg, dns1035LabelFmt, "my-name", "abc-123"))
        }
        return errs
}
```

Reply:
tbh, i did not know that k8s does it that way, I will use it, thank you!